src/Evolution/Systems/Cce/Components/WorldtubeBoundary.hpp

```diff
@@ -15,36 +15,21 @@
 namespace Cce {
 
 /*!
- * \brief Component that supplies CCE worldtube boundary data.
- *
- * \details The \ref DataBoxGroup associated with the worldtube boundary
- * component contains a data manager (e.g. `WorldtubeDataManager`) linked to
- * an H5 file. The data manager handles buffering and interpolating to desired
- * target time points when requested via the simple action
- * `BoundaryComputeAndSendToEvolution`, at which point it will send the required
- * collection of boundary quantities to the identified 'CharacteristicEvolution'
- * component. It is assumed that the simple action
- * `BoundaryComputeAndSendToEvolution` will only be called during the
- * `Evolve` phase.
- *
- * Uses const global tags:
- * - `Tags::LMax`
- *
- * `Metavariables` must contain:
- * - the `enum` `Phase` with at least `Initialization` and `Evolve` phases.
- * - a type alias `cce_boundary_communication_tags` for the set of tags to send
- * from the worldtube to the evolution component. This will typically be
- * `Cce::Tags::characteristic_worldtube_boundary_tags<Tags::BoundaryValue>`.
+ * \brief Generic base class for components that supply CCE worldtube boundary
+ * data. See class specializations for specific worldtube boundary components.
  */
-template <class Metavariables>
-struct H5WorldtubeBoundary {
-  using end_time_tag = Tags::EndTimeFromFile;
+template <typename WorldtubeComponent>
+struct WorldtubeComponentBase;
+
+template <class Metavariables, template <typename> class WorldtubeComponent>
```

Comment:
just make the second template parameter to be `typename WorldtubeComponent`.  The template template parameter does not gain you anything (in fact it makes it less flexible)